text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+# OpenTelemetry Sandbox
+
+The OpenTelemetry Sandbox is a place under OpenTelemetry's governance where the community can collaborate and experiment on projects that still aren't mature enough to be accepted as part of the official OpenTelemetry project.
+
+## Motivation
+
+Over the history of OpenTelemetry, there have been situations where people came to our community proposing interesting ideas to be adopted. There has also been vendors offering code donations to the project, some of which are now mostly unmaintained. 
+
+As a possible solution to this, this OTEP proposes a new GitHub organization, [opentelemetry-sandbox](https://github.com/opentelemetry-sandbox). This organization will host projects until there's confidence that they have a healthy community behind them. They would also serve as a neutral place for the community to conduct experiments.
+
+The advantage of a sandbox organization is that OpenTelemetry's governance rules can be applied, making sure it’s an inclusive place for people to collaborate while keeping the reputation of the OpenTelemetry project as a whole untouched, given that it would be clear that OpenTelemetry doesn’t officially support projects within the sandbox.
+
+There is a desire, but not an expectation, that projects will be moved from the sandbox as an official SIG or incorporated into an existing SIG. There’s also no expectation that the OpenTelemetry project will provide resources to the sandbox project, like extra GitHub CI minutes or Zoom meeting rooms, although we might evaluate individual requests.
+
+Projects in the sandbox should leave it within one year, either by being merged into existing SIGs, being accepted as independent SIGs, or by being archived.
```

Comment:
this directly contradicts the previous paragraph. Even CNCF does not have this "leave in one year"(but it does have annual reviews).

Reply:
What I'm trying to convey is that it's OK to experiment and to have failed experiments. So, there's a hope, but not expectation that they will "graduate". I added the following, do you think it's closer to what I mean?

> Realistically, we know that experiments might get dropped.

Reply:
I think what you want is an annual review with possible outcomes being:
  * continue as sandbox for another year
  * graduate to official OTEL
  * abandon project and archive the repo
  * transition to another org if there is a better fit

This is not the same as "should leave within one year", which is often unrealistic, nor has a strong justification.

Reply:
The concern I had was in having projects in the sandbox indefinitely, without giving enough motivation to move forward. Annual reviews would solve this, as the project might be folded after one year when there's not enough activity.

Reply:
If code is made as part of this organization and is open-source, should we have an official "transition" step here, or just assume that people will fork if they want to continue working on that? Would "transitioning to another org" mean that we'd legally transfer the assets there (name, copyright of the code)? If the project is named "opentelemetry-data-visualizer", are we OK in having another entity own it with this name?

I would keep this off for now, and just let people fork the archived repository.